client/ayon_applications/utils.py

```diff
@@ -619,12 +619,18 @@ def prepare_context_environments(
 
     data["env"]["AYON_HOST_NAME"] = app.host_name
 
+    # Check if folder and task are present in context
     if not folder_entity or not task_entity:
-        # QUESTION replace with log.info and skip workfile discovery?
-        # - technically it should be possible to launch host without context
-        raise ApplicationLaunchFailed(
-            "Host launch require folder and task context."
+        missing_context = []
+        if not folder_entity:
+            missing_context.append('folder')
+        if not task_entity:
+            missing_context.append('task')
+        log.warning(
+            f"Missing {', '.join(missing_context)} from context. "
+            "Skipping workfile preparation."
         )
```

Comment:
```suggestion
        log.warning(
            "Task is not set. Skipping workfile preparation."
        )
```

Reply:
Shall I raise an error when a folder is not provided? I can imagine working without tasks but not without folders ATM.

Reply:
Not sure about that. Considering what actually happens here in this function, it should be just:
```python
if not task_entity:
    log.warning(
        "Task is not set. Skipping workfile preparation."
    )
    return
```

Reply:
You don't have to, the information has no value, only what matters is if is set task.

The message should be `"Task is not set. Skipping workfile preparation."` and it doesn't matter if folder is or is not set, the task is the important bit.